File: app/src/main/java/studio/mandysa/music/logic/network/ServiceCreator.java
package studio.mandysa.music.logic.network;

import mandysax.anna2.Anna2;

/**
 * @author liuxiaoliu66
 */
public class ServiceCreator {
    private static final Anna2 ANNA = Anna2.build().baseUrl("http://cloud-music.pl-fe.cn/");

    public static <T> T create(Class<T> clazz) {
        return ANNA.create(clazz);
    }
}

File: app/src/main/java/studio/mandysa/music/logic/network/ServiceCreator.kt
package studio.mandysa.music.logic.network

import mandysax.anna2.Anna2

/**
 * @author liuxiaoliu66
 */
object ServiceCreator {
    private val ANNA = Anna2.build().baseUrl("http://cloud-music.pl-fe.cn/")
    fun <T> create(clazz: Class<T>): T {
        return ANNA.create(clazz)
    }
}